### ISSUE TITLE
The relationship between tumour budding, the tumour microenvironment and survival in patients with primary operable colorectal cancer

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#22]
## TL;DR

Tumour budding has been reported to reflect invasiveness, metastasis and unfavourable prognosis in colorectal cancer. The aim of the study was to examine the relationship between tumour budding and clinicopathological characteristics, tumour microenvironment and survival in patients with primary operable colorectal cancer.

### Paper Link

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4947698/

### Author/Institution

van Wyk HC (University of Glasgow)

## Overview

The presence of tumour budding was associated with elements of the tumour microenvironment and was an independent adverse prognostic factor in patients with primary operable colorectal cancer. Specifically high tumour budding stratifies effectively the prognostic value of tumour stage, venous invasion and GMS. Taken together, tumour budding should be assessed routinely in patients with primary operable colorectal cancer.

## Methods

colorectal cancer. survival analysis

## Cite

van Wyk HC, Park JH, Edwards J, Horgan PG, McMillan DC, Going JJ. The relationship between tumour budding, the tumour microenvironment and survival in patients with primary operable colorectal cancer. Br J Cancer. 2016 Jul 12;115(2):156-63. doi: 10.1038/bjc.2016.173. Epub 2016 Jun 14. PMID: 27299960; PMCID: PMC4947698.
